### PR TITLE
Update README.md with drush examples

### PR DESCRIPTION
**Pull request by DigitLib**
After merger PR #140 added drush support. Need to add it in read me. After approve or correct need to add also in wiki page for 7.x-1.12.
Fixed links to islandora https pages.

**JIRA Ticket**: https://jira.duraspace.org/browse/ISLANDORA-2280

* Other Relevant Links (Google Groups discussion, related pull requests, Release pull requests, etc.)

# What does this Pull Request do?

Add new Drush explanation

# What's new?
A in-depth description of the changes made by this PR. Technical details and possible side effects.

Example:
* Changes x feature to such that y
* Added x
* Removed y

# How should this be tested?

A description of what steps someone could take to:
* Reproduce the problem you are fixing (if applicable)
* Test that the Pull Request does what is intended.
* Please be as detailed as possible.
* Good testing instructions help get your PR completed faster.

# Additional Notes:
Any additional information that you think would be helpful when reviewing this PR.

Example:
* Does this change require documentation to be updated? 
* Does this change add any new dependencies? 
* Does this change require any other modifications to be made to the repository (ie. Regeneration activity, etc.)? 
* Could this change impact execution of existing code?

# Interested parties
Tag (@ mention) interested parties or, if unsure, @Islandora/7-x-1-x-committers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,20 @@ In order to comply with XACML restrictions placed on objects, a hook is used to
 
 ![image](https://cloud.githubusercontent.com/assets/2371345/9816201/d7e9a1e6-5871-11e5-90a0-51381eaf8fcb.png)
 
+### Drush 
+
+Apply XACML policy to target object. For example:
```
Comment (bondjimbond) [inline README.md]: Make this a header rather than the first sentence in a paragraph. Basically structure I'm thinking is like this...

**Apply a XACML policy to one target object**

To add policy.xml to object islandora:57: `drush -v --user=1 islandora_xacml_editor_apply_policy --policy=/tmp/policy.xml --pid=islandora:57`

To apply this policy to islandora:57 and all child objects, add the `--traversal` option.

**Force XACML inheritance to child objects**

To apply the XACML policy from islandora:root to its children:

`drush -v --user=1 islandora_xacml_editor_force_policy_inheritance --pid=islandora:root`

To apply this policy only to immediate children, use the `--shallow_traversal` option.

File: README.md [hunk 2]
```diff
@@ -35,6 +35,20 @@ In order to comply with XACML restrictions placed on objects, a hook is used to
...
+Apply policy.xml to \'islandora:57\' and use traversal to target child objects. 
+drush -v --user=1 islandora_xacml_editor_apply_policy --policy=/tmp/policy.xml --pid=islandora:57 --traversal
```
Comment (bondjimbond) [inline README.md]: Use `markdown` to differentiate the drush command from the rest of the text

**Review by bondjimbond — CHANGES_REQUESTED** [review]
Thanks for this contribution! It's valuable to include the Drush options, but it's a bit hard to read. Please reorganize it a bit: use headers, put the Drush commands in code markdown so they're easier to spot, and break up the paragraphs so they're more task-oriented.

File: README.md (hunk 1 above)
Comment (bondjimbond) [reply README.md]: Regarding the `--traversal` and `--shallow_traversal` options:
- For `islandora_xacml_editor_apply_policy` I assume if you don't include one of these, it applies to the target object only, correct? And is `--shallow_traversal` an option for this command?
- For `islandora_xacml_editor_force_policy_inheritance`, what happens if you don't include either option? Is there a default? If so, what is it? Or is one of these options required?
- Are there any other options that aren't listed?

Comment (DigitLib) [reply README.md]: Every options I sow is here
https://github.com/Islandora/islandora_xacml_editor/blob/7.x/islandora_xacml_editor.drush.inc#L19

I made an info from that lines....

Comment (DigitLib) [reply README.md]: @bondjimbond I corrected with your suggestions see it 
https://github.com/Islandora/islandora_xacml_editor/pull/142/files

Comment (bondjimbond) [reply README.md]: Good stuff. Just two small notes:
- For consistency, remove the "For example" on line 40
- Line 54: PIDs don't have datastreams, objects do.. Maybe reword for clarity - "The target object must have a POLICY datastream."

Comment (DigitLib) [reply README.md]: @bondjimbond Done! Tnx for clearings and suggestion.
When this add is verified and merged I'll add it on a wiki page also...

Comment (bondjimbond) [reply README.md]: Looking good! One more picky note: can you remove the period in the header on line 40? After that I can approve. :)

Comment (DigitLib) [reply README.md]: Removed! Check it...

**Review by bondjimbond — APPROVED** [review]

**Comment by bondjimbond** [comment]
Don't forget a parallel pull to 7.x-1.12

**Comment by DigitLib** [comment]
@bondjimbond Added to 7.x-1.12

**Review by bondjimbond — APPROVED** [review]
👍

**Comment by bondjimbond** [comment]
Thanks, @DigitLib. I'll merge both PRs in a couple of hours.

Just a note for future pull requests: When filling out the PR Template (your initial comment when making the pull request), please delete the filler text and any headers you aren't using. This will make it easier to read and work with.

Thanks for the contributions! Sorry if I seem picky in my reviewing. Clarity is very important to me. :)

**Comment by DigitLib** [comment]
@bondjimbond Great!! Tnx for advice I'll clean all field not needed. 
I agree for clarity 100%  and thank You for help to do this, working on documentation is also hard working.
